### ISSUE TITLE
Searchbar: Redundant voiceover due to label and placeholder.

### ISSUE THREAD

**Issue by StevenDufresne** [WordPress/wporg-parent-2021#62]
See: WordPress/wporg-showcase-2022#25 for more information.

**Comment by ryelle** [comment]
Adding more context [from slack (thread)](https://wordpress.slack.com/archives/C02RP4X03/p1663279430556099):

> "Search lesson plans" has a label of "Search for". This results in reading that's "Search landmark. Search for: Search lesson plans". It's very verbose. Would be better if there was just a visible label with "Search lesson plans" and no placeholder.

The issue is that we have a placeholder, basically. The label exists, but is hidden. All the new search fields have been styled to use this pattern, which will create the verbose output (a hidden label and a placeholder). Additionally, while I don't think this technically fails WCAG, the placeholder works as the visual label and it disappears as soon as you start typing.

<img width="359" alt="" src="https://user-images.githubusercontent.com/541093/207087876-0d215a25-ca6d-44ab-88e2-01139532cf54.png">

Learn fixed this by making the label visual & removing the placeholder.

<img width="331" alt="" src="https://user-images.githubusercontent.com/541093/207087991-10696b4c-78a6-46b9-bce1-c420f29531c4.png">
